### ISSUE TITLE
no module named carla_config

### ISSUE THREAD

**Issue by unhandyandy** [falkTX/Carla#764]
Just cloned and built it.  When I try to start it I get

```
$  source/frontend/carla
Traceback (most recent call last):
  File "source/frontend/carla", line 22, in <module>
    from carla_host import *
  File "/home/dabrowsa/Music/Carla/source/frontend/carla_host.py", line 22, in <module>
    from carla_config import *
ImportError: No module named 'carla_config'

```

**Comment by falkTX** [comment]
Please post the output of `make features`
Most likely you are missing the stuff needed to start the front-end.
See https://github.com/falkTX/Carla/blob/master/INSTALL.md

**Comment by unhandyandy** [comment]
```
$ make features
---> Main features 
Front-End:     YES (Using Qt5)
LV2 plugin:    YES
VST plugin:    YES
Link support:  YES
OSC support:   YES
Binary detect: YES

---> Engine drivers 
JACK:        YES
ALSA:        YES
PulseAudio:  NO  [Missing PulseAudio]
CoreAudio:   NO  [MacOS only]
ASIO:        NO  [Windows only]
DirectSound: NO  [Windows only]
WASAPI:      NO  [Windows only]

---> Plugin formats: 
Internal: YES
LADSPA:   YES
DSSI:     YES
LV2:      YES
VST:      YES (with UI)

---> LV2 UI toolkit support: 
External: YES (direct)
Gtk2:     YES (bridge)
Gtk3:     YES (bridge)
Qt4:      YES (bridge)
Qt5:      YES (bridge)
X11:      YES (direct+bridge)
Cocoa:    NO  [MacOS only]
Windows:  NO  [Windows only]

---> File formats: 
Basic: NO [libsndfile missing]
Extra: NO [FFmpeg missing or too new]
SF2/3: NO [FluidSynth missing or too old]
SFZ:   YES

---> Internal plugins: 
Basic Plugins:    YES
Carla-Patchbay:   YES
Carla-Rack:       YES
External Plugins: NO

$ ./source/carla
bash: ./source/carla: No such file or directory
$ ./source/frontend/carla
Traceback (most recent call last):
  File "./source/frontend/carla", line 22, in <module>
    from carla_host import *
  File "/home/dabrowsa/Music/Carla/source/frontend/carla_host.py", line 41, in <module>
    import patchcanvas
  File "/home/dabrowsa/Music/Carla/source/frontend/patchcanvas.py", line 34, in <module>
    from PyQt5.QtSvg import QGraphicsSvgItem, QSvgRenderer
ImportError: No module named 'PyQt5.QtSvg'

```

**Comment by falkTX** [comment]
You are now missing pyqt5 svg package..
I mean, this should be a bit obvious to you, no need to keep posting things without saying a word at all and just expect a fix for random stuff, that seems a bit impersonal and rude to be honest.

Install that, and carla frontend will run for you.

**Comment by unhandyandy** [comment]
Sorry, I wasn't trying to be rude, but you asked me to post the output of `make features` so I did, and moreover it didn't show any dependencies missing. Doesn't that mean there's a bug in `make features`?

**Comment by falkTX** [comment]
There is no bug, it is just qt5 that splits the svg stuff into a separate package, so you need to install that separately.
There is nothing we can do to automatically detect this (apart from running carla itself), so I closed this issue as you only have a final point to fix, but purely on your side.

**Comment by unhandyandy** [comment]
OK, `make features` is only intended to check for compilation deps, that makes sense.

Sorry I got on your nerves, but I was a little exasperated trying to chase down the dep indicated by

```
Front-End:     NO  [Missing PyQt]
```

There are about 2 dozen packages with PyQt in their names, I'd prefer not to have to install all of them (but I did).

Btw, there seems to be a mistake in the INSTALL.md:

[$ ./source/carla]

fails because there no such file.
